frontend/packages/core/src/AppLayout/header.tsx

```diff
@@ -68,20 +55,22 @@ const Header: React.FC = () => {
 
   return (
     <>
-      <AppBar position="static" color="secondary">
+      <AppBar position="static" color="secondary" elevation={0}>
         <Toolbar>
           <MenuButton onClick={openDrawer} edge="start" color="primary" data-qa="menuBtn">
             <MenuIcon />
           </MenuButton>
```

Comment:
this hamburger menu does not exist anymore afaict, it's a fixed sidebar

Reply:
yep! going to work on the navbar changes in a separate PR so kept it for now